modules/help_editor/php/module.class.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * This serves as a hint to LORIS that this module is a real module.
+ * It does nothing but implement the module class in the module's namespace.
+ *
+ * PHP Version 5
+ *
+ * @category Behavioural
+ * @package  Main
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris-Trunk/
```

Comment:
This URL is wrong (I suspect this isn't the only place, but I just noticed it now.. we can probably let it slide and then do a batch cleanup with sed after all this cleanup is done.)

Reply:
Counts in the /modules firectories :
      6 https://www.github.com/aces/CCNA/
      7 https://www.github.com/aces/Loris
     45 https://www.github.com/aces/Loris/
     77 https://www.github.com/aces/Loris-Trunk/

I will fix this one here and the other in a second PR